local_planner/src/nodes/local_planner_node.cpp

```diff
@@ -338,16 +337,16 @@ void LocalPlannerNode::cmdLoopCallback(const ros::TimerEvent& event) {
   ros::Duration since_last_cloud = now - last_wp_time_;
   ros::Duration since_start = now - start_time_;
 
-  checkFailsafe(since_last_cloud, since_start, planner_is_healthy_, hover_);
+  checkFailsafe(since_last_cloud, since_start, hover_);
 
   // If planner is not running, update planner info and get last results
   updatePlanner();
 
   // send waypoint
-  if (!never_run_ && planner_is_healthy_) {
+  if (companion_state_ == MAV_STATE::MAV_STATE_ACTIVE)
     calculateWaypoints(hover_);
-    if (!hover_) setSystemStatus(MAV_STATE::MAV_STATE_ACTIVE);
-  } else {
+
+  if (!never_run_) {
```

Comment:
do we need to unsubscribe from the camera_info topic in every iteration?

Reply:
No, you are right. I will try and fix this